celery/app/task.py

```diff
@@ -880,6 +880,11 @@ def replace(self, sig):
                 link=self.request.callbacks,
                 link_error=self.request.errbacks,
             )
+        elif isinstance(sig, chain):
```

Comment:
Should this be `_chain` instead to handle signatures reconstructed by `.from_dict()`?

Reply:
The other one above uses group instead of _group which it existed (maybe I was wrong) so I figured I should follow suite.

Reply:
I was wrong.
This is now fixed.